templates/solutions/ai/edge.html

```diff
@@ -0,0 +1,272 @@
+        </p>
+        <hr class="is-muted" />
+        <p class="p-section--shallow">
+          <a href="https://microk8s.io/">Learn more about MicroK8s&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Bring the power of the cloud to the edge</h2>
+      </div>
+      <div class="col">
+        <p>Process large data volumes and run complex ML models in the field by serving compute directly to the edge.</p>
+        <p>
+          MicroClouds are small-scale, open source clouds consisting of low-footprint clusters of compute nodes with distributed storage and secure networking. They are lightweight, low-touch, and designed for repeatable and reliable remote deployments.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="https://ubuntu.com/engage/guide-to-micro-clouds">CTO's guide to MicroCloud for the edge&nbsp;&rsaquo;</a>
+        </p>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Learn more about edge and AI</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <p class="p-heading--5">
+            <a href="https://ubuntu.com/engage/run-ai-at-scale">Run AI at scale</a>
+          </p>
+          <p>Read our whitepaper to learn how to build a performant ML Stack with NVIDIA DGX and Kubeflow.</p>
+        </div>
+        <div class="p-section--shallow">
+          <hr class="is-muted" />
+          <p class="p-heading--5">
+            <a href="https://ubuntu.com/blog/machine-learning-security-risks">An overview of machine learning security risks</a>
+          </p>
+          <p>Discover the most prominent security threats facing AI projects and how to mitigate them.</p>
+        </div>
+        <div class="p-section--shallow">
+          <hr class="is-muted" />
+          <p class="p-heading--5">
+            <a href="https://ubuntu.com/engage/exploring-edge-computing-in-automotive">Edge computing in automotive</a>
+          </p>
+          <p>
+            Watch the webinar on how edge computing is impacting the automotive industry, including the rise of autonomous vehicles.
+          </p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <hr class="is-fixed-width" />
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>Send us a message to discuss your AI needs</h2>
+      <p class="p-heading--2">
+        <a href="/contact-us" aria-controls="ai-modal">Contact us&nbsp;&rsaquo;</a>
+      </p>
+    </div>
+  </section>
+
+  <script defer src="{{ versioned_static('js/modals.js') }}"></script>
+
+  {% with %}
+    {% set returnURL = "https://canonical.com/solutions/ai#contact-form-success" %}
```

Comment:
```suggestion
    {% set returnURL = "https://canonical.com/solutions/ai/edge#contact-form-success" %}
```
Should this not return to the same page?

Reply:
It very much should, nice catch!